### PR TITLE
Enable JUnit to run Augmented Tests

### PR DESCRIPTION
**Pull request by gabrielsuch**

### CODE REVIEW

File: src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java [hunk 1]
```diff
@@ -24,7 +25,14 @@
  */
 public class CommandLineArguments {
 
+    static class ParserProperties {
```
Comment (SenhorCastor) [inline src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java]: Can you reuse CAPABILITES from PRopertiesModule and defin SAUCE there? (So all are defined in one place)

File: src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java [hunk 2]
```diff
@@ -24,7 +25,14 @@
...
+        public static final String CAPABILITIES = "CAPABILITIES";
+        public static final String SAUCE = "SAUCE";
+    }
+
     private static Class<?> hackClass;
+
+    public static final String DEFAULT_CONFIG = "conf/augmented.properties";
```
Comment (SenhorCastor) [inline src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java]: same, move it to PropertiesMOdule.

File: src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java (hunk 1 above)
Comment (gabrielsuch) [inline src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java]: Sure!

File: src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java [hunk 3]
```diff
@@ -14,6 +14,7 @@
 import java.nio.file.Paths;
```
Comment (SenhorCastor) [inline src/main/java/com/salesforceiq/augmenteddriver/util/CommandLineArguments.java]: This no longer initialized only for Command Line, rename it properly

**Comment by SenhorCastor** [comment]
LGTM! You can merge. Welcome to the commiters club!

**Pull request merged**